psalm-baseline.xml

```diff
@@ -269,9 +269,6 @@
     <PossiblyNullOperand>
       <code><![CDATA[$this->port]]></code>
     </PossiblyNullOperand>
-    <UndefinedAttributeClass>
-      <code>SensitiveParameter</code>
-    </UndefinedAttributeClass>
```

Comment:
To be reverted here, otherwise all good :+1:

Reply:
Oof. I did the opposite on rebase:  reverted other changes and kept this one